profiles/arch/arm/armv6j/use.force

```diff
@@ -0,0 +1,10 @@
+# Force armv6 specific USE flags
+cpu_flags_arm_v4
+cpu_flags_arm_v5
+cpu_flags_arm_v6
+
+cpu_flags_arm_edsp
+cpu_flags_arm_thumb
+cpu_flags_arm_thumb2
```

Comment:
Thumb seems to be mandatory since armv6, https://en.wikipedia.org/wiki/List_of_ARM_microarchitectures -- note that it's present even on plain ARMv6. About Thumb2, you're probably right.